packages/frontend-shared/src/utils/isAllowedFeature.ts

```diff
@@ -0,0 +1,104 @@
+import interval from 'human-interval'
+import { BannerIds } from '@packages/types'
+import type { LoginConnectStore } from '../store/login-connect-store'
+
+const bannerIds = {
+  isLoggedOut: BannerIds.ACI_082022_LOGIN,
+  needsOrgConnect: BannerIds.ACI_082022_CREATE_ORG,
+  needsProjectConnect: BannerIds.ACI_082022_CONNECT_PROJECT,
+  needsRecordedRun: BannerIds.ACI_082022_RECORD,
+}
+
+/**
+ *
+ * @param eventTime - timestamp of an event
+ * @param waitTime - time to compare with, such as `1 day`, `20 minutes`, etc, to be parsed by `human-interval` package
+ */
+const minTimeSinceEvent = (eventTime: number | undefined, waitTime: string) => {
+  if (!eventTime) {
+    // if there is no event time, the event has never been recorded
+    // so the "time since" is infinite, return true here
+    return true
+  }
```

Comment:
This feels backwards to me - I would think this function would enforce a period of time since an event occurred, thus if the event hasn't happened then we don't satisfy the condition

Reply:
Maybe this is a problem with the name of the function (or the comment). We don't require that one event happened before the other, we only care that, if the first did happen, Cypress doesn't fire something else without a cooldown period. So this is the logic we want.

But I get why it would seem like a generic function with this name would work the other way. I sort of think, in context, reading the rules, this is OK with the current name, but the comment should be updated. I'll think on a better name though (and would love a suggestion if you have one).